### PR TITLE
Use vets json schema fixtures for pension pdf tests

### PR DESCRIPTION
**Pull request by scottsdevelopment**
## Summary

- Add option to use the vets json schema examples in the pdf fill
- Use fixtures from the vets json schema

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#76004

## Testing done

- [x] *New code is covered by unit tests*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -152,7 +152,7 @@ gem 'swagger-blocks'
 # POSIX systems should have this already, so we're not going to bring it in on other platforms
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
```
Comment (mjknight50) [inline Gemfile:155]: Is this for testing purposes only?  Normally we would leave this pointing to master?

File: Gemfile.lock [hunk 2]
```diff
@@ -73,8 +73,8 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 9f735b78cb350ce891c50434b24e35b5fd5b8e54
-  branch: master
+  revision: bb06b010a79d5526a4becbc826b2cf07fc8a8335
+  branch: feature/pension-fixtures
```
Comment (mjknight50) [inline Gemfile.lock:77]: Same question as above

File: Gemfile.lock [hunk 3]
```diff
@@ -73,7 +73,7 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 9f735b78cb350ce891c50434b24e35b5fd5b8e54
+  revision: 11a82ea935fe45e1cea7b6b6af87842c31605a09
   branch: master
   specs:
     vets_json_schema (20.37.1)
```
Comment (danlim715) [inline Gemfile.lock:79]: this should be pointing to version 20.38.0

**Review by danlim715 — APPROVED** [review]

**Comment by ryan-mcneil** [comment]
@scottsdevelopment Just wanted to note that I got a little trigger happy and and updated your branch to master (so when you make changes, you should pull first). Sorry about that. Note that there's a linting error preventing this merge. Once it's clean I'll give you the ✅

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**